--- a/node_modules/myqnode/myQ.js	2016-03-27 16:01:16.000000000 -0700
+++ b/node_modules/myqnode/myQ.js	2016-03-27 16:01:53.000000000 -0700
@@ -217,7 +217,7 @@
 		//Closes the garage door with the given device id.
 		closeDoor : function(username, password, deviceId) {
 			return myQImpl.getConnection(username, password).then(function(respObj){
-				return myQImpl.setDeviceStatus(deviceId,2);
+				return myQImpl.setDeviceStatus(deviceId,0);
 			}).then(function(respObj){
 				return respObj.ReturnCode;
 			});
